docker-compose.yml

```
@@ -2,6 +2,7 @@ version: "3.9"
 
 x-common-env: &config_env
   DB_HOST: db
+  LOG_LEVEL: ${LOG_LEVEL}
```

Não tem diferença comportamental, é mais para ref, aqui daria pra ser
```suggestion
  LOG_LEVEL:
```

Se quiser ver um pouco sobre, as refs na doc do docker:
- https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers
- https://docs.docker.com/compose/compose-file/compose-file-v3/#environment